COC_PyBay.md

```diff
@@ -0,0 +1,76 @@
+
+This isn't an exhaustive list of things that you must do, or can't do. Rather, take it in the spirit in which it's intended. It's a guide to make it easier to enrich all of us and the technical communities in which we participate, and which we represent.
+The principles
+
+ - **Be friendly and welcoming.** We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to, members of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+ - **Be respectful.** Not all of us will agree all the time, but disagreement is no excuse for poor behavior and poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. It’s important to remember that a community where people feel uncomfortable or threatened is not a productive one. Members of the Python community should be respectful when dealing with other members as well as with people outside the community.
+ - **Be careful in the words that you choose.** We are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants, individually or as a group. Harassment and other exclusionary behavior aren't acceptable. This includes, but is not limited to:
+
+     - Violent threats or language directed against another person or yourself.
+     - Discriminatory jokes and language.
+     - Posting sexually explicit or violent material.
+     - Posting (or threatening to post) other people's personally identifying information ("doxing").
+     - Personal insults, especially those using racist or sexist terms.
+     - Unwelcome sexual attention.
+     - Deliberate intimidation, stalking, or following.
+     - Harassing photography or recording, or photographing or recording people who have explicitly declined to be recorded.
+     - Inappropriate physical contact.
+     - Advocating for, or encouraging, any of the above behavior.
+     - Repeated harassment of others. In general, if someone asks you to stop, then stop.
+
+##Where does the code of conduct apply
+
+This code of conduct applies to all spaces managed by PyBay. This includes:
+
+ - Conferences (including social events and peripheral activities, both virtual and in-person)
+ - Unconferences and sprints
+ - Meetups
+ - Workshops
+ - Presentation materials used in talks or sessions
+ - Slack
+ - Mailing lists
+ - GitHub
+ - Social media hashtag
```

Comment:
hashtag*s*?

Also, we put CoC on all of GitHub but only on specific social media hashtags? Does this mean that if someone is violating the CoC only on their tweets *not* tagged with `#pybay`, it's not a violation?

Reply:
its harder for us to be aware of this.

Reply:
I think it may be deferred to the platform guidelines. We can't enforce outside of community spaces, a general online harassment incident online between two attendees between the hours of the conference can't be definitively proved to be associated with the conference.